src/modules/src/position_estimator_altitude.c

```diff
@@ -100,8 +104,12 @@ static void positionEstimateInternal(state_t* estimate, const baro_t* baro, cons
       filteredZ = (state->estAlphaAsl       ) * state->estimatedZ +
                   (1.0f - state->estAlphaAsl) * baro->asl;
     }
-    // Use asl as base and add velocity changes.
-    state->estimatedZ = filteredZ + (state->velocityFactor * state->velocityZ * dt);
+    #ifdef IMPROVED_BARO_Z_HOLD
+      state->estimatedZ = filteredZ;
+    #else
```

Comment:
Maybe remove the original formula entirely? Or is it used in other situations?
The velocity changes do not seem to help with the estimation...